tests/toolkit/test_vector.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+import os
+from tempfile import TemporaryDirectory
+from unittest.mock import MagicMock
+
+import pytest
+from goose.toolkit.vector import VectorToolkit
+
+@pytest.fixture
+def temp_dir():
+    with TemporaryDirectory() as temp_dir:
+        yield Path(temp_dir)
+
+@pytest.fixture
+def vector_toolkit():
+    return VectorToolkit(notifier=MagicMock())
+
+def test_query_vector_db_creates_db(temp_dir, vector_toolkit):
```

Comment:
can use `tmp_path` directly instead of `temp_dir`.

`tmp_path` is the built-in fixture in pytest. https://docs.pytest.org/en/latest/how-to/tmp_path.html#tmp-path